tests/conftest.py

```diff
@@ -302,37 +387,64 @@ def fixture_config_file(config_dir: Path, configs: dict) -> Path:
 @pytest.fixture(scope="function", name="sample_vcf")
 def fixture_sample_vcf(vcf_dir: Path) -> Path:
     """Return the path to a vcf file."""
-    return vcf_dir / "example.vcf"
+    return Path(vcf_dir, "example.vcf")
+
+
+@pytest.fixture(scope="function", name="spring_file_1")
+def fixture_spring_file_1(sequencing_files_dir: Path) -> Path:
+    """Return the path to a SPRING file."""
+    return Path(sequencing_files_dir, "lane1.spring")
+
+
+@pytest.fixture(scope="function", name="spring_file_2")
+def fixture_spring_file_2(sequencing_files_dir: Path) -> Path:
+    """Return the path to a SPRING file."""
```

Comment:
```suggestion
    """Return the path to a SPRING file."""
```

Further up spring is used in lower case, we should keep one or the other